source/ways-of-working/standards/flyway-database-migrations.html.md.erb

```diff
@@ -0,0 +1,126 @@
+---
+title: Flyway database migration
+last_reviewed_on: 2022-03-17
+review_in: 6 months
+weight: 10
```

Comment:
I don't think this page is in the right place, it's in a standards section but it's a guide, should either be in:

'Guides' or 'Starting a new component'

Reply:
Moved